### PR TITLE
Update sharanvasoya.json

### PR DESCRIPTION
**Pull request by Vasoyasharan**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by triach-rold — APPROVED** [review]
Website is reachable and complete. Nice job.

**Review by MaskDuck — DISMISSED** [review]

File: domains/sharanvasoya.json [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-        "URL": "https://vasoyasharan.github.io/MyPortfolio/"
-    }
+  "description": "Sharan's website",
+  "repo": "https://github.com/Vasoyasharan/MyPortfolio",
+  "owner": {
+    "username": "Vasoyasharan",
+    "email": "sharanvasoya@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://vasoyasharan.github.io/MyPortfolio/"
```
[the remark this answers is not in the document]
Comment (Vasoyasharan) [reply domains/sharanvasoya.json]: is CNAME ok now?

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/sharanvasoya.json [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
-        "URL": "https://vasoyasharan.github.io/MyPortfolio/"
-    }
+  "description": "Sharan's website",
+  "repo": "https://github.com/Vasoyasharan/MyPortfolio",
+  "owner": {
+    "username": "Vasoyasharan",
+    "email": "sharanvasoya@gmail.com"
+  },
+  "record": {
+    "CNAME": "Vasoyasharan.in"
```
[the remark this answers is not in the document]
Comment (Vasoyasharan) [reply domains/sharanvasoya.json]: is it reachable now?

**Review by iostpa — APPROVED** [review]
seems fine to me now

**Pull request merged**